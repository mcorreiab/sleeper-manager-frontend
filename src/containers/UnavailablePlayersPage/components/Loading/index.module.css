.container {
  background-color: #1f2027;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  height: 100vh;
}
