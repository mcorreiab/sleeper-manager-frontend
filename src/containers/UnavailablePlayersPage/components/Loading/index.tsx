import { Player } from "@lottiefiles/react-lottie-player";
import classNames from "classnames";

const loading: React.FunctionComponent = () => (
  <div
    className={classNames(
      "bg-background-color",
      "flex",
      "flex-col",
      "items-center",
      "justify-center",
      "h-full",
      "text-[2rem]",
      "font-light"
    )}
  >
    <Player
      autoplay
      loop
      src="https://assets9.lottiefiles.com/packages/lf20_NXDoNy.json"
    />
    <h1 className="text-sm-lightwhite">Loading...</h1>
  </div>
);

export default loading;
