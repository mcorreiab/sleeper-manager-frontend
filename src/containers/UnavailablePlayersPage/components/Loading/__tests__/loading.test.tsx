import renderer from "react-test-renderer";
import Loading from "../index";

it("Should render loading without props", () => {
  const tree = renderer.create(<Loading />).toJSON();
  expect(tree).toMatchSnapshot();
});
