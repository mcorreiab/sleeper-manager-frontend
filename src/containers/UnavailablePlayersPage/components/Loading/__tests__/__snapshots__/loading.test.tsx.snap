// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render loading without props 1`] = `
<div
  className="bg-background-color flex flex-col items-center justify-center h-full text-[2rem] font-light"
>
  <div
    className="lf-player-container"
  >
    <div
      id="lottie"
      style={
        Object {
          "background": "transparent",
          "margin": "0 auto",
          "outline": "none",
          "overflow": "hidden",
        }
      }
    />
  </div>
  <h1
    className="text-sm-lightwhite"
  >
    Loading...
  </h1>
</div>
`;
